app/assets/javascripts/directives/validation/validation_status.js

```diff
@@ -1,16 +1,23 @@
 ManageIQ.angular.app.directive('validationStatus', ['$rootScope', function($rootScope) {
   return {
     require: 'ngModel',
-    link: function (scope, elem, attrs, ctrl) {
-      ctrl.$validators.validationRequired = function (modelValue, viewValue) {
+    link: function(scope, elem, attrs, ctrl) {
+      ctrl.$validators.validationRequired = function(modelValue, viewValue) {
         if (viewValue !== undefined && viewValue === true) {
-          scope.postValidationModelRegistry(attrs.prefix);
+          if (! scope.postValidationModelRegistry) {
+            /**
+            * awfull hack to fix the emsCommonFormController to work with angular authCredentials component
+            * TODO break the emsCommonFormController into multiple controllers/components and fix this directive
+            */
+            scope.$parent.$parent.$parent.postValidationModelRegistry(attrs.prefix);
```

Comment:
`scope.$parent.$parent.$parent` ?? OMG

@himdel 
While I applaud @Hyperkid123 for attempting this huge feat, I think this PR is a little too premature.
At the very least, we should have first converted all controllers that use `auth credentials` to `controllerAs`.
I think that would be the right course of action here.

@Hyperkid123 I am wondering if you can hit the pause button here in this PR, and wait a while until all the controllers in question are first converted to `controllerAs`.